LICENSE

```diff
@@ -1,21 +1,3 @@
-MIT License
```

Comment:
I think you have changed the license file mistakenly instead of README file.

Reply:
Reverted back :man_facepalming: